src/components/Button/ButtonFunction/index.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { LayoutType } from '../ButtonContent'
+import { ButtonFunction, clickMe } from '.'
+import renderWithTheme from '../../../testing/renderWithTheme'
+
+test('renders button with function and layout line', () => {
```

Comment:
This is duplicate of previous button test. Here it should test that action happens when user clicks on a button. :wink: